website/static/js/addonSettings.js

```diff
@@ -17,6 +17,7 @@ var ExternalAccount = function(data) {
     self.name = data.display_name;
     self.id = data.id;
     self.profileUrl = data.profile_url;
+    self.provider_name = data.provider_name;
```

Comment:
> Use lowerCamelCase for variables, properties and function names

https://github.com/felixge/node-style-guide#use-lowercamelcase-for-variables-properties-and-function-names